src/server.cc

```diff
@@ -167,6 +167,8 @@ Status Server::Start() {
     }
   });
 
+  LOG(INFO) << "Ready to accept connections\n";
```

Comment:
```suggestion
  LOG(INFO) << "Ready to accept connections";
```
newline is automatically applied in glog

Reply:
Updated.